core/org.wso2.carbon.ui/src/main/resources/web/admin/login.jsp

```diff
@@ -56,6 +60,22 @@ if (CharacterEncoder.getSafeText(request.getParameter("skipLoginPage"))!=null){
 	return;
 }
 
+AdminAdvisoryBannerDTO adminAdvisoryBannerConfig = null;
+Boolean enableBanner = null;
+String bannerContent = null;
+String backendServerURL = CarbonUIUtil.getServerURL(config.getServletContext(), session);
+ConfigurationContext configContext = (ConfigurationContext) config.getServletContext()
+                .getAttribute(CarbonConstants.CONFIGURATION_CONTEXT);
+AdminAdvisoryBannerClient client = new AdminAdvisoryBannerClient(backendServerURL, configContext);
+
+try {
+    adminAdvisoryBannerConfig = client.loadBannerConfig();
+    enableBanner = adminAdvisoryBannerConfig.getEnableBanner();
+    bannerContent = adminAdvisoryBannerConfig.getBannerContent();
+
+} catch (Exception e) {
+    e.printStackTrace();
```

Comment:
shall we extract this out to a method? you can use declaration syntax (<%! ... %>)